<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>865</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1122</width>
    <height>865</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1122</width>
    <height>865</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FacultyWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="usernameLE">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>355</y>
      <width>325</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordLE">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>485</y>
      <width>325</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="proceedFW">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>650</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(170, 170, 127)</string>
    </property>
    <property name="text">
     <string>Proceed</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-220</x>
      <y>0</y>
      <width>1401</width>
      <height>871</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/back/BACKGROUND.png) 0 0 0 0 stretch stretch</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>210</y>
       <width>450</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Leelawadee</family>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#aaaa7f;&quot;&gt;Faculty Window&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="usernameLabel">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>340</y>
       <width>220</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Leelawadee</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; color:#c6f9fd;&quot;&gt;Username:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="passwordLabel_2">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>480</y>
       <width>220</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Leelawadee</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; color:#cdf4f2;&quot;&gt;Password:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="back_fw">
     <property name="geometry">
      <rect>
       <x>1100</x>
       <y>90</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>1</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Leelawadee</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(170, 170, 127)</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>usernameLE</zorder>
   <zorder>passwordLE</zorder>
   <zorder>proceedFW</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
